cluster/src/main/java/com/hartwig/pipeline/sbpapi/FileResponse.java

```diff
@@ -0,0 +1,8 @@
+package com.hartwig.pipeline.sbpapi;
+
+public class FileResponse {
+    public int id;
+
+    public FileResponse() {
```

Comment:
empty constructor and non-final mutable field? looks like a job for immutables

Reply:
Indeed. Was actually looking for a way to inline it but will switch to an Immutable.